native/dlabad.go

```diff
@@ -0,0 +1,11 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+func (impl Implementation) Dlabad(small, large float64) {
```

Comment:
It was unclear to me how closely to follow the original code. I'm all for deleting code if we think that's okay.